saskatoon/member/admin.py

```diff
@@ -164,6 +166,7 @@ def has_password(self, user):
                     'is_staff',
                     'is_superuser',
                     'has_temporary_password',
+                    'agreed_terms',
```

Comment:
I think you want this field in `fieldsets`, not `add_fieldsets`, essentially if for some reason we create a new user from admin panel then `agreed_terms` get set to False, and then later on we want to be able as admin to toggle the value if needed